Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -314,6 +315,25 @@ namespace xr
 
             return task;
         }
+
+        /**
+         * Compare two poses and determine if they are approximately equal.
```

Comment:
Instead of an operator, should this have a name that clarifies it's an approximate comparison, such as `ApproxEqual()`?

Reply:
Will do